authentication/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="Looking for Frieda | Log in" content="Looking for Frieda log in page" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.6.0/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../dist/css/index.css" />
+    <title>Authentication page</title>
+  </head>  
+  
+  <body class="body-login">
+    <nav class="navbar navbar-expand-lg navbar-custom pt-5 pb-5">
+      <img class="logo" src="/images/logo.png" alt="logo of Looking for Frieda">
+    </nav>
+    
+    <!-- form validation -->
+    <div class="container">
+      <div class="row justify-content-center align-items-center min-vh-100">
+        <div class="col-12col-md-8">
```

Comment:
There is a missing space here, check out offsets: https://getbootstrap.com/docs/5.3/layout/columns/#offsetting-columns